build/admission/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17.13-alpine3.16 AS builder
+FROM golang:1.19.12-alpine3.17 AS builder
```

Comment:
@Shelley-BaoYue would it be possible to update all **golang** `1.19.12` container image versions to `alpine3.18`?

https://hub.docker.com/layers/library/golang/1.19.12-alpine3.18/images/sha256-085007a33ce53bd5130bb5f485cad6c6f0fbfa203907cb5b05e575bc37a409aa?context=explore

Reply:
> @Shelley-BaoYue would it be possible to update all **golang** `1.19.12` container image versions to `alpine3.18`?
> 
> https://hub.docker.com/layers/library/golang/1.19.12-alpine3.18/images/sha256-085007a33ce53bd5130bb5f485cad6c6f0fbfa203907cb5b05e575bc37a409aa?context=explore

The decision to choose alpine3.17 is based on the following considerations: Firstly, 3.17 is compatible with golang 1.19. In the future, when upgrading the golang version to 1.20 or higher, we can further upgrade to 3.18. Secondly, opting for a slightly older version ensures greater stability. However, if you believe that 3.18 is more suitable, we welcome you to discuss it at any time.